examples/redis-vectorstore-example/go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/tmc/langchaingo/examples/redis-vectorstore-example
+
+go 1.22.0
+
+toolchain go1.22.1
+
+// NOTE: remove the following line to use the official (rather than local development) version
+replace github.com/tmc/langchaingo => ../..
```

Comment:
you can use a go.work file to manage this locally. A doc suggesting a dev workflow would be super useful.

Reply:
rm example temporarily，because of #782